apps/cms/pages/templates/a4_candy_cms_pages/home_page.html

```diff
@@ -2,14 +2,21 @@
 {% load i18n wagtailcore_tags wagtailimages_tags %}
 
 {% block content %}
-    <div class="hompage__wrapper p-3">
-        <div class="row homepage-header">
-            <div class="col-lg-6">
-                <h1 class="homepage-header__teaser ml-md-4">{{ page.teaser|richtext }}</h1>
+    <div class="homepage__wrapper p-3">
+        <div class="homepage-header">
+            <div class="row">
+                <div class="col-lg-6">
+                    <h1 class="homepage-header__teaser ml-md-4">{{ page.teaser|richtext }}</h1>
+                </div>
+                <div class="col-lg-6 homepage-header__image-wrapper">
+                    {% image page.image original as image %}
+                    <img src="{{ image.url }}" alt="{{ image.alt }}" class="homepage-header__image" />
+                </div>
             </div>
-            <div class="col-lg-6 homepage-header__image-wrapper">
-                {% image page.image width-770 as image %}
-                <img src="{{ image.url }}" alt="{{ image.alt }}" class="homepage-header__image" />
+            <div class="row">
```

Comment:
Wait! Now I am confused again. Sorry! Shouldn't this be everywhere else than on the homepage? And be directly after the header?